docker-compose.yml

```diff
@@ -26,6 +26,11 @@ services:
       AWS_ACCESS_KEY_ID: longkey
       AWS_SECRET_ACCESS_KEY: verysecretkey
 
+  message-bus:
+    image: redis:5.0.5-alpine
+    healthcheck:
+      test: ["CMD", "nc", "-z", "localhost", "6379"]
```

Comment:
Would it be better to use `redis-cli PING` ?

Reply:
I guess it would be less likely to change in the case that the redis image changes and they remove linux tools such as `nc`. Will update to use redis-cli ping.

Reply:
updated.